lux/action/default.py

```diff
@@ -19,8 +19,7 @@ def register_default_actions():
     lux.config.register_action("correlation", correlation, no_vis)
     lux.config.register_action("distribution", univariate, no_vis, "quantitative")
     lux.config.register_action("occurrence", univariate, no_vis, "nominal")
-    lux.config.register_action("temporal", univariate, no_vis, "temporal")
-    lux.config.register_action("geographical", geomap, no_vis, "geographical")
```

Comment:
@micahtyong I'm not sure if you are depending on this, but if so, let's not remove this import.

Reply:
Yeah, I am, thanks for the catch! @jerrysong1324

Reply:
Thanks for the catch!